lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -583,6 +583,8 @@ def process(self, wet_bulb_temperature):
             wet_bulb_temperature_integral (iris.cube.Cube):
                 Cube of wet bulb temperature integral (Kelvin-metres).
         """
+        # Touch the data to ensure it is not lazy
+        wet_bulb_temperature.data
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Statement seems to have no effect](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4683550)

Reply:
# pylint: disable=pointless-statement